src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -340,5 +340,6 @@
   </Target>
 
   <!-- This helps collect crash dumps and requires python installed -->
-  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" Condition="'$(EnableDumpling)' == 'true' OR '$(EnableCloudTest)' == 'true'" />
+  <!-- Enable if dumpling is not explicitly disabled in cloud test scenarios, or if it is enabled explicitly in local scenarios -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" Condition="'$(EnableDumpling)' == 'true' OR ('$(EnableDumpling)' != 'false' AND '$(EnableCloudTest)' == 'true')" />
```

Comment:
I think I would like to keep the import purely set based on the EnableDumplying property and somewhere above do this.

```
<EnableDumpling Condition="'$(EnableDumplying)' == '' and '$(EnableCloudTest)' == 'true'">true</EnableDumpling>
```